python/lsst/rubintv/production/rubinTv.py

```diff
@@ -825,3 +850,201 @@ def beat(self, ensure=False, customFlatlinePeriod=None):
         if (elapsed >= self.uploadPeriod) or ensure or customFlatlinePeriod:
             if self.uploader.uploadHeartbeat(self.handle, forecast):  # returns True on successful upload
                 self.lastUpload = now  # only reset this if the upload was successful
+
+
+class CalibrateCcdRunner():
+    """Class for running CharacterizeImageTask and CalibrateTasks on images.
+
+    Runs these tasks and writes shards with various measured quantities for
+    upload to the table.
+    """
+    def __init__(self, outputRoot, *, doRaise=False, embargo=False):
+        self.dataProduct = 'quickLookExp'
+        self.uploader = Uploader()
+        self.butler = makeDefaultLatissButler(embargo=embargo, writeable=True)
+        self.log = _LOG.getChild("monitorChannel")
+        self.channel = 'auxtel_characterizeImage'
+        self.watcher = Watcher(self.dataProduct, self.channel)
+        self.fig = plt.figure(figsize=(12, 12))
+        self.doRaise = doRaise
+        self.shardsDir = os.path.join(outputRoot, 'shards')
+        self.outputRunName = "LATISS/runs/quickLook/1"
```

Comment:
This needs taking from somewhere more central. DM-37272 exists for this, so please add a `TODO: DM-37272` to remind me/help me find it.